tests/types/test-date32.py

```diff
@@ -113,6 +114,58 @@ def test_date32_minmax():
     assert dt.Type.date32.max == 2146764179
 
 
+#-------------------------------------------------------------------------------
+# Statistics
+#-------------------------------------------------------------------------------
+
+def test_date32_minmax():
+    src = [None,
+           d(2000, 10, 18),
+           d(2010, 11, 13),
+           d(2020, 2, 29),
+           None]
+    DT = dt.Frame(src)
+    assert DT.min1() == d(2000, 10, 18)
+    assert DT.max1() == d(2020, 2, 29)
+    assert DT.countna1() == 2
+    assert_equals(DT.min(), dt.Frame([d(2000, 10, 18)]))
+    assert_equals(DT.max(), dt.Frame([d(2020, 2, 29)]))
+    assert_equals(DT.countna(), dt.Frame([2]/dt.int64))
+
+
+def test_date32_na_stats():
+    src = [None,
+           d(2000, 10, 18),
+           d(2010, 11, 13),
+           d(2020, 2, 29),
+           None]
+    DT = dt.Frame(src)
+    assert DT.sd1() == None
+    assert DT.sum1() == None
```

Comment:
`is None` is better than `== None` (otherwise python linter will complain).

Reply:
Yeah, it's not the first time I forget to use `is` for `None`. Everytime I correct it, I find several other places we use `==` in the code. May be there should be an automated tool for this correction in the editor :)